src/main/java/org/jenkinsci/plugins/DependencyCheck/tools/DependencyCheckInstallation.java

```diff
@@ -61,36 +62,37 @@ public DependencyCheckInstallation forEnvironment(EnvVars environment) {
         return new DependencyCheckInstallation(getName(), environment.expand(getHome()), getProperties().toList());
     }
 
+    @Override
+    public void buildEnvVars(EnvVars env) {
+        if("".equals(env.get("PATH+DependencyCheck", "")))
+            env.put("PATH+DependencyCheck", getHome() + "/dependency-check/bin");
+    }
+
     @Override
     public DependencyCheckInstallation forNode(@Nonnull Node node, TaskListener log) throws IOException, InterruptedException {
         return new DependencyCheckInstallation(getName(), translateFor(node, log), getProperties().toList());
     }
 
-    public String getExecutable(@Nonnull Launcher launcher) throws IOException, InterruptedException {
-        final VirtualChannel channel = launcher.getChannel();
-        return channel == null ? null : channel.call(new FindExecutableCallable(getHome()));
+    @SuppressFBWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE")
+    public String getExecutable(Launcher launcher) throws IOException, InterruptedException {
+        return launcher.getChannel().call(new MasterToSlaveCallable<String, IOException>() {
```

Comment:
this implementation cause log on jenkins console becasuse of use of anynoums class
https://www.jenkins.io/doc/developer/security/remoting-callables/#minimal-safe-implementations